koala/ast/graph.py

```diff
@@ -152,39 +159,45 @@ def evaluate(self,cell,is_addr=True):
             try:
                 # print '->', cell
                 cell = self.cellmap[cell]
+
             except:
-                print 'Empty cell at '+ cell
-                return []
-            
+                # print 'Empty cell at '+ cell
+                return None
+
         # no formula, fixed value
-        if not cell.formula or cell.value != None:
+        if not cell.formula or not cell.always_eval and cell.value != None:
```

Comment:
Or maybe you're thinking about something like this:
`if (not cell.formula or not cell.always_eval) and cell.value != None:`

This would mean previous code should have been `if not cell.formula and cell.value != None:`, but i don't believe this is necessary, since if the cell is not a formula, its value should never be `None` Indeed, it's an entry of the tree, which never changes.

What's your opinion ?